.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,65 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: input
+    id: terraform-version
+    attributes:
+      label: Terraform Version
+      description: What version of terraform are you running? `terraform -v`
+      placeholder: Terraform v1.0.7 on darwin_amd64
+    validations:
+      required: true
+  - type: input
+    id: provider-version
+    attributes:
+      label: Equinix Provider Version
+      description: What version of our provider are you running?
+      placeholder: version = "1.17.0"
+    validations:
+      required: true
+  - type: textarea
+    id: resources
+    attributes:
+      label: Effected Terraform Resources
+      description: Equinix resouces causing/effected by the bug
+      placeholder: |
+        equinix_metal_device
+        equinix_fabric_connection
+  - type: textarea
+    id: config
+    attributes:
+      label: Terraform Config Files
+      description: If your config files are too long for a short code snippet please use GitHub Gists
+  - type: textarea
+    id: debug
+    attributes:
+      label: Debug Output
+      description: To generate debug output run `terraform apply` with `TF_PROVIDER_LOG=DEBUG` and `PACKNGO_DEBUG=1`. Please use GitHub Gists if the output is long
```

Comment:
There may be other variables to consider here. We could also caution against submitting client_id, client_secret, auth_token, authorization headers, and x-auth-token headers